src/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -50,78 +50,66 @@ export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingMod
     >
       <TextContent>
         <Text component={TextVariants.p}>
-          Booting from an ISO takes a few minutes. It also depends heavily on the hardware and
-          network configuration. However, if you have verified that your host machine is powered on,
-          DHCP is enabled and yet the host does not appear in the inventory list, then login to the
-          host over SSH to troubleshoot. For that, you would need to know your machine IP-address.
-          You can also use a console such as BMC or virtual machine console.
+          To troubleshoot host discovery issues, complete the following steps:
         </Text>
+        <Text component={TextVariants.h2}>Verify that your host machine is powered on</Text>
+        <Text component={TextVariants.p}>
+          Note: Boot time depends on several factors such as your hardware and network
+          configuration, and if you are booting from an ISO.
+        </Text>
+        <Text component={TextVariants.h2}>Verify if DHCP is enabled</Text>
         <Text component={TextVariants.h2}>SSH into your machine</Text>
-        <PrismCode code={`ssh core@<machine-ip>`} />
         <Text component={TextVariants.p}>
-          The SSH key provided when generating the discovery ISO is used to authenticate the user.
-          It should not be required to provide the password.
+          Verify that you can access your host machine using SSH, or a console such as BMC or
+          virtual machine console.  In the CLI, enter the following command:
+        </Text>
+        <PrismCode code="ssh core@<machine-ip>" />
+        <Text component={TextVariants.p}>
+          NOTE: Authentication is provided by the discovery ISO, therefore when you access your host
+          using SSH, a password is not required.
         </Text>
         <Text component={TextVariants.h2}>
-          Verify that the discovery agent is running correctly
+          Verify that the discovery agent is running with the correct parameters
         </Text>
-        <Text component={TextVariants.p}>Execute</Text>
         <PrismCode code={`ps -ef | grep agent`} />
-        <Text component={TextVariants.p}>
-          and see whether the agent runs with correct parameters. The result should look similar to
-          this:
-        </Text>
+        <Text component={TextVariants.p}>The output displays the following:</Text>
         <PrismCode
-          code={`root        1342       1  0 09:56 ?        00:00:00 /usr/local/bin/agent --host 192.168.39.162 --port 30956 --cluster-id a8142e14-8bfe-46bd-bfbd-4bd1a126892b core        2063    2003  0 10:41 pts/0    00:00:00 grep --color=auto agent`}
+          code={`root        1775       1  0 11:19 ?        00:00:00 /usr/local/bin/agent --url http://10.19.130.1:6000 --cluster-id f1063d85-3099-4d7b-ade3-269142184044 --agent-version quay.io/ocpmetal/assisted-installer-agent:latest --insecure=true
```

Comment:
should be https:// and probably api.openshift.com or something?

Reply:
Fixed, thanks